packages/ccd-js-gen/src/cli.ts

```diff
@@ -13,6 +13,8 @@ type Options = {
     outDir: string;
     /** The output file types for the generated code */
     outputType: lib.OutputOptions;
+    /** Generate `// @ts-nocheck` annotations in each file. Disabled by default. */
```

Comment:
I would write `False` instead of `Disabled` but not feeling strongly about it.
```suggestion
    /** Generate `// @ts-nocheck` annotations in each file. False by default. */
```

Reply:
For this line, I have just removed it, since it is not optional in the type.